test/compilable/scope.d

```diff
@@ -165,11 +165,21 @@ void demangle() @safe
     {
         const(char)[] s;
 
+        @safe:
         @property bool empty() const { return !s.length; }
 
-        @property const(char)[] front() const return { return s; }
+        @property const(char)[] front() const return
+        {
+            immutable i = indexOfDot();
```

Comment:
The build failure in phobos relied on this method because it changes the semantic order.